### PR TITLE
refactor(stepper content): surf1307

### PR DESCRIPTION
**Pull request by lalithkarikelli**
JIRA: SURF-1307

### LGTM's
* [x] Dev LGTM
* [x] Dev LGTM
* [x] Design LGTM

### CODE REVIEW

File: src/scss/components/accordion/_stepper.scss [hunk 1]
```diff
@@ -99,4 +100,8 @@ hx-accordion.beta-hxStepper {
       margin-left: 0.5rem;
     }
   }
+
+  .beta-hxContainerStyle {
```
Comment (100stacks) [inline src/scss/components/accordion/_stepper.scss:104]: Try moving this style into a `<div>` block:

```scss
div {
...
```

We may have to tweak later.

File: src/scss/components/accordion/_stepper.scss [hunk 2]
```diff
@@ -99,4 +100,8 @@ hx-accordion.beta-hxStepper {
       margin-left: 0.5rem;
     }
   }
+
```
Comment (100stacks) [inline src/scss/components/accordion/_stepper.scss:103]: Please change all `.beta-hxStepXXX` classes above to `.hxStepXXX`.

Comment (100stacks) [inline src/scss/components/accordion/_stepper.scss:103]: Please change `.beta-hxSubmitButtons` to `.hxStepButton`.

File: src/scss/components/accordion/_stepper.scss [hunk 3]
```diff
@@ -99,4 +100,8 @@ hx-accordion.beta-hxStepper {
...
+  .beta-hxContainerStyle {
+    margin-left: 3rem;
+  }
 }//.beta-hxStepper
```
Comment (100stacks) [inline src/scss/components/accordion/_stepper.scss:107]: Please remove `beta-` from the comment.

File: docs/components/stepper/index.html [hunk 4]
```diff
@@ -32,7 +30,7 @@ <h2 id="basic-stepper">Basic Stepper</h2>
               <span class="beta-hxStepLabel">Account</span>
               <span class="beta-hxStepValue">Acme Corp Inc.</span>
             </header>
-            <div class="hxBox hxMd">
+            <div class="hxBox hxMd beta-hxContainerStyle">
```
Comment (100stacks) [inline docs/components/stepper/index.html:33]: We shouldn't need the extra class.  Please remove.

File: docs/components/stepper/index.html [hunk 5]
```diff
@@ -49,7 +47,7 @@ <h2 id="basic-stepper">Basic Stepper</h2>
               <span class="beta-hxStepLabel">Topic</span>
               <span class="beta-hxStepValue">Technical Incident</span>
             </header>
-            <div class="hxBox hxMd">
+            <div class="hxBox hxMd beta-hxContainerStyle">
```
Comment (100stacks) [inline docs/components/stepper/index.html:50]: We shouldn't need the extra class.  Please remove.

File: docs/components/stepper/index.html [hunk 6]
```diff
@@ -67,7 +65,7 @@ <h2 id="basic-stepper">Basic Stepper</h2>
               <span class="beta-hxStepLabel">Issue Details</span>
               <span class="beta-hxStepValue"></span>
             </header>
-            <div class="hxBox hxMd">
+            <div class="hxBox hxMd beta-hxContainerStyle">
```
Comment (100stacks) [inline docs/components/stepper/index.html:68]: We shouldn't need the extra class.  Please remove.

File: docs/components/stepper/index.html [hunk 7]
```diff
@@ -92,7 +90,7 @@ <h2 id="basic-stepper">Basic Stepper</h2>
                 <span class="beta-hxStepLabel">Account</span>
                 <span class="beta-hxStepValue">Acme Corp Inc.</span>
               </header>
-              <div class="hxBox hxMd">
+              <div class="hxBox hxMd beta-hxContainerStyle">
```
Comment (100stacks) [inline docs/components/stepper/index.html:93]: We shouldn't need the extra class.  Please remove.

File: docs/components/stepper/index.html [hunk 8]
```diff
@@ -32,7 +30,7 @@ <h2 id="basic-stepper">Basic Stepper</h2>
               <span class="beta-hxStepLabel">Account</span>
```
Comment (100stacks) [inline docs/components/stepper/index.html:30]: Please update all of these `beta` classes.

File: docs/components/stepper/index.html [hunk 9]
```diff
@@ -32,7 +30,7 @@ <h2 id="basic-stepper">Basic Stepper</h2>
...
               <span class="beta-hxStepValue">Acme Corp Inc.</span>
```
Comment (100stacks) [inline docs/components/stepper/index.html:31]: Please update all of these `beta` classes.

File: docs/components/stepper/index.html [hunk 10]
```diff
@@ -49,7 +47,7 @@ <h2 id="basic-stepper">Basic Stepper</h2>
               <span class="beta-hxStepLabel">Topic</span>
```
Comment (100stacks) [inline docs/components/stepper/index.html:47]: Please update all of these `beta` classes.

File: docs/components/stepper/index.html [hunk 11]
```diff
@@ -49,7 +47,7 @@ <h2 id="basic-stepper">Basic Stepper</h2>
...
               <span class="beta-hxStepValue">Technical Incident</span>
```
Comment (100stacks) [inline docs/components/stepper/index.html:48]: Please update all of these `beta` classes.

File: docs/components/stepper/index.html [hunk 12]
```diff
@@ -67,7 +65,7 @@ <h2 id="basic-stepper">Basic Stepper</h2>
               <span class="beta-hxStepLabel">Issue Details</span>
```
Comment (100stacks) [inline docs/components/stepper/index.html:65]: Please update all of these `beta` classes.

File: docs/components/stepper/index.html [hunk 13]
```diff
@@ -67,7 +65,7 @@ <h2 id="basic-stepper">Basic Stepper</h2>
...
               <span class="beta-hxStepValue"></span>
```
Comment (100stacks) [inline docs/components/stepper/index.html:66]: Please update all of these `beta` classes.

File: docs/components/stepper/index.html [hunk 14]
```diff
@@ -1,10 +1,8 @@
 ---
 title: Stepper
-beta: true
 also:
-  components/accordion: Accordion
-  elements/hx-accordion: <hx-accordion>
-  elements/hx-accordion-panel: <hx-accordion-panel>
```
Comment (100stacks) [inline docs/components/stepper/index.html:7]: Please revert these changes.

File: docs/components/stepper/index.html [hunk 15]
```diff
@@ -1,10 +1,8 @@
...
+  components/stepper: Stepper
+  elements/hx-stepper: <hx-accordion>
```
Comment (100stacks) [inline docs/components/stepper/index.html:5]: Not needed.  Please remove.

File: docs/_data/nav.json5 [hunk 16]
```diff
@@ -100,6 +101,7 @@
             { label: '<hx-select>', path: 'hx-select' },
             { label: '<hx-select-control>', path: 'hx-select-control' },
             { label: '<hx-status>', path: 'hx-status' },
+            { label: '<hx-stepper>', path: 'hx-stepper' },
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply docs/_data/nav.json5:104]: Changes have been done, please review.

File: docs/elements/hx-stepper/index.html [hunk 17]
```diff
@@ -0,0 +1,112 @@
+---
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply docs/elements/hx-stepper/index.html:1]: Changes have been done, please review.

File: src/scss/components/accordion/_stepper.scss [hunk 18]
```diff
@@ -8,6 +8,7 @@ hx-accordion.beta-hxStepper {
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/accordion/_stepper.scss:8]: Changes have been done, please review.

[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/accordion/_stepper.scss:8]: Changes have been done, please review.

File: src/elements/hx-accordion-panel/_shadow.html [hunk 19]
```diff
@@ -1,11 +1,11 @@
-<button type="button" id="hxToggle" aria-controls="body" aria-expanded="false">
+<div id="hxToggle" aria-controls="body" aria-expanded="false">
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/elements/hx-accordion-panel/_shadow.html:1]: Changes have been done, please review.

File: src/elements/hx-accordion-panel/_shadow.html [hunk 20]
```diff
@@ -1,11 +1,11 @@
...
   <div class="header">
     <span class="header__content">
       <slot name="header"></slot>
     </span>
     <hx-icon class="header__icon" type="angle-down"></hx-icon>
   </div>
-</button>
+</div>
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/elements/hx-accordion-panel/_shadow.html:8]: Changes have been done, please review.

File: src/scss/components/accordion/_stepper.scss (hunk 1 above)
Comment (lalithkarikelli) [reply src/scss/components/accordion/_stepper.scss:104]: Changes have been done, please review.

File: src/scss/components/accordion/_stepper.scss (hunk 2 above)
Comment (lalithkarikelli) [reply src/scss/components/accordion/_stepper.scss:103]: Changes have been done, please review.

File: src/scss/components/accordion/_stepper.scss (hunk 3 above)
Comment (lalithkarikelli) [reply src/scss/components/accordion/_stepper.scss:107]: Changes have been done, please review.

File: docs/components/stepper/index.html (hunk 4 above)
Comment (lalithkarikelli) [reply docs/components/stepper/index.html:33]: Changes have been done, please review.

File: docs/components/stepper/index.html (hunk 5 above)
Comment (lalithkarikelli) [reply docs/components/stepper/index.html:50]: Changes have been done, please review.

File: docs/components/stepper/index.html (hunk 6 above)
Comment (lalithkarikelli) [reply docs/components/stepper/index.html:68]: Changes have been done, please review.

File: docs/components/stepper/index.html (hunk 7 above)
Comment (lalithkarikelli) [reply docs/components/stepper/index.html:93]: Changes have been done, please review.

File: docs/components/stepper/index.html (hunk 8 above)
Comment (lalithkarikelli) [reply docs/components/stepper/index.html:30]: Changes have been done, please review.

File: docs/components/stepper/index.html (hunk 9 above)
Comment (lalithkarikelli) [reply docs/components/stepper/index.html:31]: Changes have been done, please review.

File: docs/components/stepper/index.html (hunk 10 above)
Comment (lalithkarikelli) [reply docs/components/stepper/index.html:47]: Changes have been done, please review.

File: docs/components/stepper/index.html (hunk 11 above)
Comment (lalithkarikelli) [reply docs/components/stepper/index.html:48]: Changes have been done, please review.

File: docs/components/stepper/index.html (hunk 12 above)
Comment (lalithkarikelli) [reply docs/components/stepper/index.html:65]: Changes have been done, please review.

File: docs/components/stepper/index.html (hunk 13 above)
Comment (lalithkarikelli) [reply docs/components/stepper/index.html:66]: Changes have been done, please review.

File: docs/components/stepper/index.html (hunk 14 above)
Comment (lalithkarikelli) [reply docs/components/stepper/index.html:7]: Changes have been done, please review.

File: docs/components/stepper/index.html (hunk 15 above)
Comment (lalithkarikelli) [reply docs/components/stepper/index.html:5]: Changes have been done, please review.

File: src/scss/components/accordion/_stepper.scss (hunk 2 above)
Comment (lalithkarikelli) [reply src/scss/components/accordion/_stepper.scss:103]: Changes have been done, please review.

File: src/scss/components/accordion/_stepper.scss (hunk 3 above)
Comment (100stacks) [reply src/scss/components/accordion/_stepper.scss:107]: This was missed.  Please removed `beta-` on Line 107:

https://github.com/HelixDesignSystem/helix-ui/pull/675/files#diff-dc0ca2e5fac9b8928afe09ccdd3b84c0R107

File: docs/components/stepper/index.html [hunk 21]
```diff
@@ -1,6 +1,5 @@
 ---
 title: Stepper
```
Comment (100stacks) [inline docs/components/stepper/index.html:2]: Please add a reference to the `Stepper` component on the `Accordion` page:

https://deploy-preview-675--helix-ui.netlify.app/components/accordion/

File: src/scss/components/accordion/_stepper.scss [hunk 22]
```diff
@@ -79,24 +80,28 @@ hx-accordion.beta-hxStepper {
...
+  .hxStepButton {
     .hxBtn + .hxBtn {
       margin-left: 0.5rem;
     }
   }
+
+  div {
+    margin-left: 3rem;
+  }
 }//.beta-hxStepper
```
Comment (100stacks) [inline src/scss/components/accordion/_stepper.scss:107]: Please remove `beta-` from the comment.

File: src/elements/hx-accordion-panel/_shadow.scss [hunk 23]
```diff
@@ -2,6 +2,7 @@
 
 #hxToggle {
   background-color: transparent;
+  outline: 0;
```
Comment (100stacks) [inline src/elements/hx-accordion-panel/_shadow.scss:5]: Good effort, though we still need to show focus when a user tabs through the Stepper. 

Please make the focus similar to http://helix.rax.io/style/focus-states.html

File: docs/components/stepper/index.html (hunk 21 above)
Comment (lalithkarikelli) [reply docs/components/stepper/index.html:2]: Changes have been done, please review.

File: src/scss/components/accordion/_stepper.scss (hunk 22 above)
Comment (lalithkarikelli) [reply src/scss/components/accordion/_stepper.scss]: Changes have been done, please review.

File: src/elements/hx-accordion-panel/_shadow.scss (hunk 23 above)
Comment (lalithkarikelli) [reply src/elements/hx-accordion-panel/_shadow.scss]: Changes have been done, please review.

File: docs/components/accordion/index.html [hunk 24]
```diff
@@ -5,6 +5,7 @@
   components/accordion/test.html: Testing - Accordion
   elements/hx-accordion: <hx-accordion>
   elements/hx-accordion-panel: <hx-accordion-panel>
+  components/stepper/index.html: Stepper
```
Comment (100stacks) [inline docs/components/accordion/index.html:8]: Our current pattern is to reference components before testing and element documentation.

Please move this above `components/accordion/test.html: Testing - Accordion`.

File: src/elements/hx-accordion-panel/_shadow.scss [hunk 25]
```diff
@@ -41,3 +41,8 @@
     display: block;
   }
 }
+
+button:focus {
+  outline: none;
+  box-shadow: 0 0 4px 0 #0e94a6;
+}
```
Comment (100stacks) [inline src/elements/hx-accordion-panel/_shadow.scss:48]: Please move this into the Light DOM styles:

https://github.com/HelixDesignSystem/helix-ui/pull/675/files#diff-dc0ca2e5fac9b8928afe09ccdd3b84c0

This provides more flexibility for styling component states.

File: docs/components/accordion/index.html (hunk 24 above)
Comment (lalithkarikelli) [reply docs/components/accordion/index.html]: Changes have been done, please review.
